src/client.ts

```diff
@@ -2628,7 +2629,7 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
      *
      * @returns the cross signing information for the user.
      */
-    public getStoredCrossSigningForUser(userId: string): CrossSigningInfo | null {
+    public getStoredCrossSigningForUser(userId: string): Promise<CrossSigningInfo | null> {
```

Comment:
this is a breaking change; we can't do this. We'll need to make a new method and deprecate the old one.

Reply:
(this will also mean you don't need to change `QRCode.ts` I think